spec/features/admin/enterprise_roles_spec.rb

```diff
@@ -170,9 +172,16 @@
     end
   end
 
-
   private
 
+  def navigate_to_enterprise_users
+    within ".side_menu" do
+      click_link "Users"
+    end
+
+    expect(page).to have_selector "table.managers"
```

Comment:
I wouldn't add this here as we're coupling the actual navigation with an assertion. As a result, in line 133 above we're adding an extra step that we don't need. That being said, I don't think line 101 should need it.

Reply:
what do you think @kristinalim ?

Reply:
@sauloperez I moved check for table presence out of the helper method. Build is green, and I moved this back to Code Review.